code/modules/surgery/organs/augments_chest.dm

```diff
@@ -5,43 +5,6 @@
 	implant_overlay = "chest_implant_overlay"
 	zone = "chest"
 
-/obj/item/organ/cyberimp/chest/nutriment
```

Comment:
if it ever comes back it'll require stuffing food into it and having it auto-feed you